docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+
+services:
+  core:
+    build: .
+    restart: always
```

Comment:
This will restart every time the computer restarts? And then run the simulation?

Reply:
Yes, exactly.

Reply:
All good - if this makes sense with docker desktop it is all good with me, no problem.